Decorator/CarDecorator/BaseCar.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Decorator
+{
+    class BaseCar : ICar
```

Comment:
Maybe we need to leave only one Text decorator

Reply:
delete